.circleci/config.yml

```diff
@@ -52,6 +52,11 @@ jobs:
           keys:
             - v1-dependencies-{ { checksum "package.json" }}
 
+      - run: |
+          echo "common --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY}" >> .bazelrc.local
+          echo "common --config=ci" >> .bazelrc.local
+          echo "test --define=APPLITOOLS_API_KEY=${APPLITOOLS_API_KEY} --define=APPLITOOLS_BATCH_ID=${CIRCLE_SHA1}" >> .bazelrc.local
```

Comment:
I think this, as-is, is problematic since it would apply those flags on _every_ `test` invocation, causing some rebuild/reanalysis since those flags weren't defined during `build`. So, either we'd need to apply these flags on `common` or leave them on the specific Android test command so that is the only place that eats the perf impact. I'm trying to think of a reason to not have it on `common`, the only thing I can think of is whether or not that would invalidate remote caching? Or maybe that'd just be on the first time and subsequent builds with that flag would be able to pull from the cache properly? Thoughts?

Reply:
I think that could also mess with the ability to share the cache between local and remote, since none of the remote builds would be sharable for pulling down to your local environment

Reply:
Moving it back to the android call might be the safest